packages/amplify_authenticator/example/integration_test/reset_password_test.dart

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+// This test follows the Amplify UI feature "sign-in-with-username"
+// https://github.com/aws-amplify/amplify-ui/blob/main/packages/e2e/features/ui/components/authenticator/sign-up-with-username.feature
+import 'package:amplify_api/amplify_api.dart';
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'config.dart';
+import 'pages/forgot_password_page.dart';
+import 'pages/sign_in_page.dart';
+import 'pages/test_utils.dart';
+import 'utils/data_utils.dart';
+import 'utils/mock_data.dart';
+
+void main() {
+  final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized()
+      as IntegrationTestWidgetsFlutterBinding;
+  // resolves issue on iOS. See: https://github.com/flutter/flutter/issues/89651
+  binding.deferFirstFrame();
+
+  final authenticator = MaterialApp(
+    home: Authenticator(
+        child: const SignOutButton(
+      key: Key('keySignOutButton'),
+    )),
+  );
+
+  group('reset-password', () {
+    // Given I'm running the example "ui/components/authenticator/reset-password.feature"
+    setUpAll(() async {
+      await loadConfiguration('ui/components/authenticator/reset-password',
+          additionalConfigs: [AmplifyAPI()]);
+    });
+
+    tearDownAll(() async {
```

Comment:
Would prefer to do this in `loadConfiguration` so that it's the first thing done in all tests.

Reply:
Good idea